src/main/webapp/assets/js/feedScript.js

```diff
@@ -901,7 +974,7 @@ document.addEventListener('keydown', function(e) {
       modalCard.focus();
       e.preventDefault();
     }
-    if (document.activeElement === modalSubmittedText) {
+    if (document.activeElement === modalSubmittedTitle) {
```

Comment:
This was solved in #83 !

Reply:
Oh! I guess I hadn't pulled `master` recently enough. I've pulled now, so it should have overwritten/not be in my changes anymore